services/static.js

```diff
@@ -237,6 +263,9 @@ Static.getStaticImage = function(config) {
   if (config.layer_id !== undefined) {
     query.layer_id = config.layer_id;
   }
+  if (config.padding !== undefined) {
```

Comment:
[question] how do you feel about adding an assertion on the values of `padding` ahead of this line? Preferably some minimal guardrails to prevent users from generating URLs that are are very obviously wrong (e.g. `padding: 'a random string of letters'`)

Reply:
👋🏽  This seems to be handled pretty well on the API side. For example: 
https://api.mapbox.com/styles/v1/mapbox/streets-v11/static/[-122,34,-121,45]/600x600?&padding=dgdf&access_token=TOKEN will have the error message of ` "Padding must be 1-4 numbers, comma-delimited."` Do you think it is still required to be handled on the SDK side too?

Reply:
👍 summarizing our chat from earlier: the existing design of the SDK seems content to not validate any parts of the options provided aside from the standard type checking coming from fusspot and let the APIs' errors speak for themselves. That sounds good to me, so we should feel comfortable leaving this without more rigorous arg checks.